src/api.js

```diff
@@ -0,0 +1,28 @@
+var url = 'https://en.wikipedia.org/w/api.php';
+
+var params = {
+  action: 'query',
+  generator: 'geosearch',
+  prop: 'coordinates|pageimages',
+  ggscoord: '37.7891838|-122.4033522',
+  format: 'json',
+};
+
+url = url + '?origin=*';
+Object.keys(params).forEach(function (key) {
+  url += '&' + key + '=' + params[key];
```

Comment:
Oh, another thing is you can make this code simpler/easier to read by using ES6 templates as so:

    url += `&${ key }=${ params[key] }`;